libgadget/tests/test_forcetree.c

```diff
@@ -36,9 +36,9 @@ order_by_type_and_key(const void *a, const void *b)
         return -1;
     if(pa->Type > pb->Type)
         return +1;
-    if(pa->Key < pb->Key)
+    if(pa->Pos[0] < pb->Pos[0])
```

Comment:
Is this ordering important? (It was ordered by key and now just by the first pos component, suggesting the order may be only for readability?)

Reply:
As I remember the sort was so that the test was more realistic: the default build order was so close to the natural tree ordering that it was unreasonably fast all the time! Maybe I can put this back as it was.